pkgs/applications/misc/cartridges/default.nix

```diff
@@ -0,0 +1,62 @@
+{ blueprint-compiler
+, desktop-file-utils
+, fetchFromGitHub
+, gobject-introspection
+, lib
+, libadwaita
+, meson
+, ninja
+, python3
+, stdenv
+, testers
+, wrapGAppsHook4
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "cartridges";
+  version = "2.0.1";
+
+  src = fetchFromGitHub {
+    owner = "kra-mo";
+    repo = "cartridges";
+    rev = "v${finalAttrs.version}";
+    sha256 = "sha256-ApPO8OhKVg4Wcp+oYHLcgLksi0muOpGlv/p0zsKxTM4=";
```

Comment:
```suggestion
    sha256 = "sha256-vtfSDyJeqsyMBkDIqcXceYZi9ODcB/YinLg5qp606zk=";
```

Reply:
thanks for catching this! i was using `nurl` and forgot to put a space between the repo url and the tag :/